utils/common/src/funTest/kotlin/SafeDeleteRecursivelyFunTest.kt

```diff
@@ -43,6 +49,27 @@ class SafeDeleteRecursivelyFunTest : WordSpec({
             dir.exists() shouldBe false
         }
 
+        "be able to delete files with non-UTF8 characters in their name" {
+            val pkg = Package.EMPTY.copy(
+                vcsProcessed = VcsInfo(
+                    VcsType.GIT,
+                    "https://github.com/oss-review-toolkit/ort-test-fork-node-dir.git",
+                    "a57c3b1b571dd91f464ae398090ba40f64ba38a2"
+                )
+            )
+
+            val nodeDir = tempdir().resolve("node-dir")
+            Git().download(pkg, nodeDir)
```

Comment:
Why did you decide for cloning a repository instead of just programatically creating the files?

(The latter would also avoid turning this into a fun test and execute faster and offline)

Reply:
I've clarified that the JVM cannot create such files, and we cannot commit such a file to this repo either.